src/validators/digit.js

```diff
@@ -0,0 +1,20 @@
+import isNil from 'ramda/src/isNil';
+import isEmpty from 'ramda/src/isEmpty';
+
+const validate = val  => {
+  if (isNil(val)) {
+    return true;
+  }
+
+  if (isEmpty(val)){
```

Comment:
Be careful 
```
_.isEmpty(0) -> true
_.isEmpty(1) -> true
```
Probably better to move the logic to cast the number to string before this if clause

```
const strValue = String(val);

if (isEmpty(val)) {
  return false;
}

const regexp = /^\d*$/g;

return regexp.test(strVal);
```